ui/components/app/nfts-items/nfts-items.js

```diff
@@ -182,13 +183,14 @@ export default function NftsItems({
                   className="nfts-items__item-wrapper"
                 >
                   <NftItem
-                    src={nftImage}
+                    src={image ? nftImage : ''}
                     alt={nftImageAlt}
                     name={name}
                     tokenId={tokenId}
                     networkName={currentChain.nickname}
                     networkSrc={currentChain.rpcPrefs?.imageUrl}
                     onClick={handleImageClick}
+                    isIpfsEnabled={Boolean(ipfsGateway)}
```

Comment:
Instead of prop drilling down the chain, should we not just use the ipfsSelector in the constituent components?